src/Task.js

```diff
@@ -0,0 +1,28 @@
+export default class Task {
+  constructor(title, description, date, priority, now, createBy) {
+    this.title = title;
+    this.description = description;
+    this.date = date;
+    this.priority = priority;
+    this.now = now;
+    this.createBy = createBy;
+  }
+
+  static getTask() {
+    let tasks;
+    if (localStorage.getItem('tasks') === null) {
+      tasks = [];
+    } else {
+      tasks = JSON.parse(localStorage.getItem('tasks'));
+    }
+
+    return tasks;
+  }
+
+  static storeTask(task) {
+    const tasks = Task.getTask();
+    tasks.push(task);
+    localStorage.setItem('tasks', JSON.stringify(tasks));
+  }
+}
+//  localStorage.clear()
```

Comment:
## Avoid committing and pushing commented out code.
- [ ] It’s better to delete **commented piece of codes**. If you need to review a previous version of your code, use `git history` instead. More on this on this [screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/no_inline_comments.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/).